torchtitan/train_spec.py

```diff
@@ -48,6 +49,7 @@ def from_model_args(cls, args: BaseModelArgs) -> nn.Module:
     [list[nn.Module], JobConfig, OptimizersContainer], OptimizersContainer
 ]
 LRSchedulersBuilder: TypeAlias = Callable[[OptimizersContainer], LRSchedulersContainer]
+LossFn: TypeAlias = Callable[[torch.Tensor, torch.Tensor], torch.Tensor]
```

Comment:
nit: to be consistent in naming
```suggestion
LossFunctionn: TypeAlias = Callable[[torch.Tensor, torch.Tensor], torch.Tensor]
```

Reply:
Done :+1: